nilearn/plotting/surface/_matplotlib_backend.py

```diff
@@ -120,30 +121,18 @@ def _adjust_plot_roi_params(params):
         params["cbar_tick_format"] = "%i"
 
 
-def _get_vertexcolor(
-    surf_map,
-    cmap,
-    norm,
-    absolute_threshold=None,
-    bg_map=None,
-    bg_on_data=None,
-    darkness=None,
-):
-    """Get the color of the vertices."""
-    if bg_map is None:
-        bg_data = np.ones(len(surf_map)) * 0.5
-        bg_vmin, bg_vmax = 0, 1
-    else:
-        bg_data = np.copy(load_surf_data(bg_map))
-
-    # scale background map if need be
+def _normalize_bg_data(bg_data):
     bg_vmin, bg_vmax = np.min(bg_data), np.max(bg_data)
     if bg_vmin < 0 or bg_vmax > 1:
         bg_norm = Normalize(vmin=bg_vmin, vmax=bg_vmax)
         bg_data = bg_norm(bg_data)
+    return bg_data
+
 
+# TODO (nilearn >= 0.13.0) remove
+def _apply_darkness(data, darkness):
```

Comment:
Here too

Reply:
this one is about to be removed (in the following months) so I think we can survive without docstring